README.md

```diff
@@ -16,20 +16,49 @@ python3 setup.py install --user
 ```
 pip3 install standup-notes
 ```
-
-## Run
+###Bash Setup
```

Comment:
Will this work if pip installed from PyPi?

Reply:
It works after I changed the version number to a lower version in the code (setup.py line 38 "3.7" to "3.6").